recipes/climtools/25.2/gh200/environments.yaml

```diff
@@ -0,0 +1,24 @@
+gcc-env:
+  compiler:
+      - toolchain: gcc
+        spec: gcc
+  mpi:
+      spec: cray-mpich@8.1.30
+      gpu: false
+  unify: true
+  specs:
+  - hdf5
+  - netcdf-c
+  - netcdf-cxx4
+  - ncview
+  - cdo
+  - nco
+  - python@3.13
+  - eccodes +tools +aec +openmp jp2k=jasper
```

Comment:
Do you need to use the same version of eccodes as was used for the `icon/25.2` uenv?
If so, see the `icon/25.2` recipe:
* it builds a specific version
* it contains a `repo` path with a custom eccodes recipe that makes some tweaks to get the specific version required for the MCH v8 stack.

Reply:
I saw that. Wanted to ask @jonasjucker, I never use eccodes myself.

Reply:
I would stick to the same eccodes version as in `icon/25.2`. Some standardization should be good.